lib/awful/tooltip.lua

```diff
@@ -441,7 +527,10 @@ function tooltip.new(args)
             hide(self)
         end
     else
-        function self.show()
+        function self.show(other, geo)
```

Comment:
I don't really understand how this works. What type of object are `other` and `geo` here?